alibi_detect/saving/validate.py

```diff
@@ -38,6 +38,7 @@ def validate_config(cfg: dict, resolved: bool = False) -> dict:
 
     # Get meta data
     meta = cfg.get('meta')
+    meta = {} if meta is None else meta
```

Comment:
Why not `meta = cfg.get('meta', {})`? Ease of reading?

Reply:
This bit was the root of the bug. The pydantic schema for `DetectorConfig` sets `meta` to `None` if it doesn't exist in the `config.toml`. Since `meta` does exist in `cfg` (but equals `None`), `cfg.get('meta', {})` will still return `None`, and then line 42 fails.

Reply:
Is this something worth leaving a comment in the code for (to prevent overzealous code "clean-up")?

Reply:
Good idea. Added a comment to this and the line in `base.py`.